best-practices/require_probes/require_probes.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
       are running or need to be restarted. A `readinessProbe` is used by services 
       and deployments to determine if the pod is ready to receive network traffic.
 spec:
-  validationFailureAction: audit
+  validationFailureAction: enforce
```

Comment:
any reason why this was changed to enforce? For most of the sample policies, we keep as audit and allow the user to set as enforce if they wish.